docs/design/slides/slides-connectors.md

```
@@ -0,0 +1,462 @@
+  y: number;     // [0, 1] in local bbox (pre-rotation)
+  angle: number; // radians; outward normal in local coords (pre-rotation)
+};
+```
+
+`angle` follows the canvas convention: `0 = +x` (right), `π/2 = +y`
+(down). Helper constants `DIR_N = -Math.PI / 2`, `DIR_E = 0`, `DIR_S =
+Math.PI / 2`, `DIR_W = Math.PI`.
+
+**Registry** (`view/canvas/connection-sites/`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)